scrape/parse_pages_to_csv.py

```diff
@@ -102,33 +104,44 @@ def _get_contact_info_from_soup(self, soup):
                 self.company_dict[key] = ''
 
     def _clean_up_crazy_whitespace_in_employees_on_site_field(self):
-        self.company_dict['Employees on Site'] = re.sub(r'\s+', '',
-                                                 self.company_dict['Employees on Site'])
+        self.company_dict['Employees'] = re.sub(r'\s+', '',
+                                                 self.company_dict['Employees'])
         self.company_dict['Phone'] = re.sub(r'\s+', '', self.company_dict['Phone'])
 
     def _get_address_from_soup(self, soup):
-        address = list(self.company_dict['Mailing Address'].split(','))
-        address1 = address[:-3]
-        self.company_dict['Address1'] = ', '.join(address1)
-        self.company_dict['Address2'] = '{0},{1} {2}'.format(address[-3], address[-2], address[-1])
-
+        try:
+            address = list(self.company_dict['Mailing Address'].split(','))
+            address1 = address[0]
+            self.company_dict['Address1'] = address1
+            # There is an extra comma in file between city and OH
+            self.company_dict['Address2'] = '{0}, {1} {2}'.format(address[-4], address[-2], address[-1])
+        except:
+            address = list(self.company_dict['Physical Address'].split(','))
+            address1 = address[0]
+            self.company_dict['Address1'] = address1
+            # There is an extra comma in file between city and OH
+            self.company_dict['Address2'] = '{0}, {1} {2}'.format(address[-5], address[-3], address[-2])
+# to edit - puts name in title field
     def _get_executive_info_from_soup(self, soup):
         executive_info = soup.select('.executivebox')
         executive_blocks = executive_info[0].select('a.vcardcontact')
 
         for index in range(5):
             try:
-                name = executive_blocks[index].find_parent('td').find_next_sibling('td').find_next_sibling('td')
+                name = executive_blocks[index].find_parent('td').find_next_sibling('td').find_next_sibling('td').find_next_sibling('td')
                 title = name.find_next_sibling('td')
-                self.company_dict['Name' + str(index + 1)] = name.get_text()
+                nameFin = name.get_text()
```

Comment:
Maybe `name = name.get_text()`

If you were to use `nameFin`, in Python it would be best to do snake-case or `name_fin`